src/Illuminate/Auth/Access/Gate.php

```diff
@@ -123,19 +128,27 @@ public function define($ability, $callback)
      * @param  string  $name
      * @param  string  $class
      * @param  array   $abilities
+     * @param  array   $options
      * @return $this
      */
-    public function resource($name, $class, array $abilities = null)
+    public function resource($name, $class, array $abilities = [], array $options = [])
     {
         $abilities = $abilities ?: [
+            'index'   => 'index',
             'view'   => 'view',
             'create' => 'create',
             'update' => 'update',
             'delete' => 'delete',
         ];
 
+        $options += ['guestable' => ['index', 'view']];
```

Comment:
Why should we default to allowing guests to access index and view? Even _if_ this PR was a good idea (which it isn't), guessable would be _manually_ provided. You're writing this PR for the community, not for yourself and your own purposes.